lib/ui/compositing/scene_builder.cc

```diff
@@ -180,34 +183,37 @@ fml::RefPtr<EngineLayer> SceneBuilder::pushPhysicalShape(const CanvasPath* path,
   return EngineLayer::MakeRetained(layer);
 }
 
+void SceneBuilder::pop() {
+  PopLayer();
+}
+
 void SceneBuilder::addRetained(fml::RefPtr<EngineLayer> retainedLayer) {
-  if (!current_layer_) {
-    return;
-  }
-  current_layer_->Add(retainedLayer->Layer());
+  AddLayer(retainedLayer->Layer());
 }
 
-void SceneBuilder::pop() {
-  if (!current_layer_) {
-    return;
-  }
-  current_layer_ = current_layer_->parent();
+void SceneBuilder::addPerformanceOverlay(uint64_t enabledOptions,
```

Comment:
Why is `addPerformanceOverlay` moved here? Maybe keep it in the old place to make the diff a little easier to read.